en/patterns/icon-title.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Icon + Title - Design System Pattern
+_description: The Icon + Title Pattern Symbols combine an Avatar with Icon or Image, Title and Body Text to display an element part of a set of components used for representation of various scenarios, such as product feature description and single step of wizard sequence.
+_keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angular, Sketch to Angular, Angular, Angular Design System, Export code from Sketch, Design Kits for Angular, Sketch HTML, Sketch to HTML, Sketch UI kits
+---
+
+## Avatar + Badge
+
+Use the Icon + Title Pattern to present a single element from a set of elements through avatar with featured icon or image, title and rich-text description. It can be used for various scenarios, such as product feature description and single step of wizard sequence.
+
+<img class="responsive-img" src="../images/icon_title.png" srcset="../images/icon_title@2x.png 2x" />
```

Comment:
You had a very good point about using a set of this pattern to define the features of a product. Make the image like that.